packages/cosmic-swingset/lib/ag-solo/vats/lib-wallet.js

```diff
@@ -1,22 +1,30 @@
 import harden from '@agoric/harden';
 import { insist } from '@agoric/ertp/util/insist';
 
+import { makeUnitOps } from '@agoric/ertp/core/unitOps';
+
+import makeObservablePurse from './observable';
+import makeStore from './store';
+import makeWeakStore from './weakStore';
+
 function mockStateChangeHandler(_newState) {
   // does nothing
 }
 
 export async function makeWallet(
   E,
-  log,
-  host,
   zoe,
   registrar,
   pursesStateChangeHandler = mockStateChangeHandler,
   inboxStateChangeHandler = mockStateChangeHandler,
 ) {
-  // Map of purses in the wallet by pet name. Assume immutable.
-  const nameToPurse = new Map();
-  const nameToAssayId = new Map();
+  const petnameToPurse = makeStore();
```

Comment:
Are we supporting petnames yet? This was called name because we didn't have support yet.
FYI @erights

Reply:
What would it mean to support petnames? The way I would think of it, a full petname system has a UI that goes out to the user, the ability to add new items with suggested (but changeable) petnames, the invariant that petnames in a category must be unique, etc. We're not there yet, but I think there's nothing wrong with starting without the UI and some of the features. We still have an API that allows the consumer of the API to name things uniquely and only for themselves, so I would call that petnames. Is there something else?